thrift/src/main/thrift/entities/film.thrift

```diff
@@ -0,0 +1,10 @@
+namespace java com.gu.contententity.thrift.entity.game
+
+struct Film {
+  1: required string title
+  2: required string year
+  3: required string imdbId
+  4: required list<string> directors
```

Comment:
This is interesting again because really we want to model a Person and especially and this point where an actor/director is of significant interest.